lib/backends/webgl/ops/reshape-packed.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {Reshape} from '../../../ops/reshape';
+import {Tensor} from '../../../tensor';
+import {ShapeUtil} from '../../../util';
+import {getGlsl} from '../glsl-source';
+import {WebGLInferenceHandler} from '../inference-handler';
+import {ProgramInfo, RunData, WebGLOperator} from '../types';
+import {TextureLayout} from '../types';
+
+import {unpackFromChannel} from './packing_utils';
+
+export class WebGLReshapePacked extends Reshape implements WebGLOperator {
+  run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
+    return inferenceHandler.run(this, inputs);
+  }
+  createProgramInfo(handler: WebGLInferenceHandler, inputs: Tensor[]): ProgramInfo {
+    if (inputs.length !== 2) {
```

Comment:
Resize -> Reshape

Reply:
Just saw this comment.. I will correct it in the next PR.